src/main/kotlin/org/rust/ide/utils/import/importUtils.kt

```diff
@@ -25,6 +25,9 @@ import org.rust.stdext.isSortedWith
 fun ImportCandidate.import(context: RsElement) = info.import(context)
 
 fun ImportInfo.import(context: RsElement) {
+    // Imports are not very important for intention preview
```

Comment:
Actually, it's not true in all cases, at least from my point of view.
For example, in the case of `Import` quick-fix it's only one meaningful change of the quick-fix. Taking into account complex structure of Ruse `use` items and sorting, it may be quite useful.

But I don't insist on fixing it in this PR, just want to change the comment to emphasise that it's temporarily state, and we want to change it in the future